exercises/practice/ocr-numbers/.docs/instructions.append.md

```diff
@@ -0,0 +1,21 @@
+# Instructions append
+
+## Implementation Notes
+
+Define a function recognizeDigit as README Step 1 except make it recognize
```

Comment:
Note that there is no `README` on the website. Use the term `instructions` instead. This is what it is called on the website and this is also what that section is called in the `README` file when you download the exercise via CLI.

Reply:
Thanks! I've tweaked the wording. Let me know what you think